shared/threat-models/ext/threat-model-grouping.model.yml

```diff
@@ -15,6 +15,7 @@ extensions:
       - ["database", "local"]
       - ["commandargs", "local"]
       - ["environment", "local"]
+      - ["stdin", "local"]
```

Comment:
Overall I think this is a good decision.

But I'm debating whether it makes more sense as a completely separate threat model kind versus a sub-kind of `file` (my understanding of how `threatModelEnabled` works is that it supports sub-kinds).

Either way I agree with @michaelnebel that this should have a change note.